Bio/Align/__init__.py

```diff
@@ -1365,6 +1365,10 @@ def __len__(self):
         return len(self.paths)
 
     def __getitem__(self, index):
+        if index < 0:
```

Comment:
To be consistent with Python indexing, if `index < 0`, you can add `len(self.paths)` to the index. If then the index is still negative, we can raise an `IndexError`. If not, we have a valid index.